src/snyk/common/configuration.ts

```diff
@@ -47,7 +47,11 @@ export class Configuration implements IConfiguration {
   }
 
   get baseURL(): string {
-    return this.isDevelopment ? 'https://deeproxy.dev.snyk.io' : this.defaultBaseURL;
+    if (this.isDevelopment) {
+      return process.env.SNYK_VSCE_DEVELOPMENT_BASE_URL ?? 'https://deeproxy.dev.snyk.io';
```

Comment:
Just thinking ahead: are we ok with envvar naming when Snyk Open Source (and its custom endpoint) will integrated too?

Reply:
Good point, made the var product-centric.